packages/webapp/src/lib/services/RegistryManager.ts

```
@@ -0,0 +1,31 @@
+  static updateUrlParam(value: string | null): void {
+    if (value) {
+      const url = new URL(window.location.href);
+      url.searchParams.set('registry', value);
+      window.history.pushState({}, '', url.toString());
+    }
+  }
+  
+  static isCustomRegistry(value: string | null): boolean {
+    return !!value && value !== REGISTRY_URL;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)